.github/workflows/actionlint.yml

```diff
@@ -0,0 +1,19 @@
+name: actionlint
+
+on:
+  push:
+    paths:
+      - .github/**
+
+permissions:
+  contents: read
+
+jobs:
+  actionlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c # v3.3.0
+      - name: "Check GitHub workflow files"
+        uses: docker://docker.mirror.hashicorp.services/rhysd/actionlint:latest
+        with:
+          args: -color -ignore SC2086
```

Comment:
I'm wary of ignoring quoting all the time, bugs resulting from unexpected variable expansion are very common.